hugo/layouts/papers/single.html

```diff
@@ -173,9 +175,11 @@ <h5 class="card-title">Abstract</h5>
         </a>
         {{ end }}
       {{ else }}
+        {{ if isset $paper "url" }}
```

Comment:
This could still be changed to `with`, but that's more of a style thing than a functional one :)

Reply:
Whoops, missed that one. Just pushed. Thank you!